packages/stream_video_push_notification/android/src/main/kotlin/io/getstream/video/flutter/stream_video_push_notification/StreamVideoPushNotificationPlugin.kt

```diff
@@ -0,0 +1,66 @@
+package io.getstream.video.flutter.stream_video_push_notification
+
+import android.app.Activity
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Build
+import io.flutter.embedding.engine.plugins.FlutterPlugin
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler
+import io.flutter.embedding.engine.plugins.activity.ActivityAware
+import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding
+import com.hiennv.flutter_callkit_incoming.CallkitNotificationManager
+
+/** StreamVideoPushNotificationPlugin */
+class StreamVideoPushNotificationPlugin: FlutterPlugin, MethodCallHandler, ActivityAware {
+  private lateinit var channel : MethodChannel
+  private var activity: Activity? = null
+  private var context: Context? = null
+  private lateinit var callkitNotificationManager: CallkitNotificationManager
+
+  override fun onAttachedToEngine(flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
+    context = flutterPluginBinding.applicationContext
+
+    channel = MethodChannel(flutterPluginBinding.binaryMessenger, "stream_video_push_notification")
+    channel.setMethodCallHandler(this)
+
+    callkitNotificationManager = CallkitNotificationManager(flutterPluginBinding.applicationContext)
+  }
+
+  override fun onMethodCall(call: MethodCall, result: MethodChannel.Result) {
+    if (call.method == "ensureFullIntentPermission") {
+      val notificationManager = context?.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+      if (Build.VERSION.SDK_INT >= 34) {
+        if (!notificationManager.canUseFullScreenIntent()) {
+          callkitNotificationManager?.requestFullIntentPermission(activity)
```

Comment:
```suggestion
          callkitNotificationManager?.requestFullScreenIntentPermission(activity)
```

Reply:
This one is missed, i guess

Reply:
this one is part of `flutter_callkit_incoming` package